urdf/CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ if(BUILD_TESTING)
   ament_cppcheck(LANGUAGE "c++")
   ament_cpplint()
   ament_lint_cmake()
-  ament_uncrustify()
+  ament_uncrustify(LANGUAGE "C++")
```

Comment:
It would be nice if the string for language was consistent with `ament_cppcheck` (ie. either both use a lowercase "c" or uppercase "C"). I guess the difference is coming from what the underlying tools expect.

Reply:
Yeah, I would rather stick to the exact naming offered by the tool.